_posts/2021-08-25-debezium-1-7-beta1-released.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  Debezium 1.7.0.Beta1 Released
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: rkerner
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
+
+This release brings https://docs.nats.io/developing-with-nats-streaming/streaming#nats-streaming-overview[NATS Streaming]
+support for Debezium Server along with many other fixes and enhancements. +
+Also this release is the first one tested with Apache Kafka 2.8.
```

Comment:
Please add the mention of Kafka 2.8 into version summary in version metadata too.

Reply:
I added "kafka" to the tags now. That's what you meant?

Reply:
@rk3rn3r I assume this is what Jiri means: https://github.com/debezium/debezium.github.io/blob/develop/_data/releases/1.7/1.7.0.Beta1.yml